app/src/main/java/com/asfoundation/wallet/nfts/domain/NFTItem.kt

```diff
@@ -0,0 +1,7 @@
+package com.asfoundation.wallet.nfts.domain
+
+import java.io.Serializable
+
+data class NFTItem(val name: String, val description: String?, val imageURL: String?,
+                   val id: String) :
+    Serializable
```

Comment:
Serializable can be on the previous line. Code style doesn't detect this, but what we generally do is put everything in the same line and let code style do the breaks in these situations.